### ISSUE TITLE
Why start from 5.2 release?

### ISSUE THREAD

**Issue by cb88** [OpenValidator/openvalidator#4]
All the commits up until the license was changed should be usable as well right?

This appears the the the last MIT licensed commit. https://github.com/nghuuphuoc/bootstrapvalidator/tree/5f3906629171b0d01c5b28f9a75c9ceef5fe1476

**Comment by emilchristensen** [comment]
Hi cb88, i was under the impression that 5.3 was where the license changed?

**Comment by cb88** [comment]
Yes but you could get all the commits up until he relicensed it in the repo... so 5.3 minus a few commits and it says it's still MIT. Currently I believe you have 5.2 plus a few commits you missed all the commits between 5.2 and when the license changed except for what you merged manually.

I'm not sure what the best way to fork from that master is... probably either 
`git reset checkout 5f3906629171b0d01c5b28f9a75c9ceef5fe1476 .`
or
`git revert --no-edit HEAD~7..`
where 7 is the number of commits to revert from the current master to get back to 5f3906629171b0d01c5b28f9a75c9ceef5fe1476 

I think the former will discard the history of the relicensed commits and the latter would revert them and keep them in the history. Or you could just cherry pick all the commits up until that point I think... not sure how to that works though no git fu isn't all that strong.

**Comment by emilchristensen** [comment]
That's a very good point. I'll merge the missing commits in :)

/Emil

**Comment by olliebennett** [comment]
Did this ever get done? If not, another option is to add [nghuuphuoc/bootstrapvalidator](https://github.com/nghuuphuoc/bootstrapvalidator) as a remote and `cherry-pick` the relevant commits from there. Would be good to close this issue.
